src/py/flwr/client/app.py

```diff
@@ -507,11 +508,15 @@ def _load_client_app() -> ClientApp:
 
                     # Don't update/change NodeState
 
+                    e_code = ErrorCode.UNKNOWN
+                    if isinstance(ex, ClientAppException):
+                        e_code = ErrorCode.CLIENT_APP_RAISED_EXCEPTION
+
                     # Create error message
                     # Reason example: "<class 'ZeroDivisionError'>:<'division by zero'>"
                     reason = str(type(ex)) + ":<'" + str(ex) + "'>"
                     reply_message = message.create_error_reply(
-                        error=Error(code=0, reason=reason)
+                        error=Error(code=e_code, reason=reason)
```

Comment:
I think it's generally Okay but I am not sure how industry guys see this. Should we mention clearly in the change log that the error will be reported to the server? I think some people may consider the exception message as privacy.

Reply:
does the `reason` carry any specific info? we could change this to just use `LOAD_CLIENT_APP_EXCEPTION` and a generic `reason`